game/scripts/npc/abilities/techies_remote_mines.txt

```diff
@@ -22,7 +22,7 @@
     // Casting
     //-------------------------------------------------------------------------------------------------------------
     "AbilityCastRange"                                    "500"
-    "AbilityCastPoint"                                    "1.0"
+    "AbilityCastPoint"                                    "0.72" //OAA
```

Comment:
I would leave this unchanged. This is something which could change the feel for techies mains.

Reply:
I have tested this out myself and .72 is a buff whilst still feeling like old techies. I will wait for other feedback as the purpose of the buff and what I've been suggested to do is to decrease castpoint.